### PR TITLE
sql: add jobs verification to TestCancelSchemaChange

### PR DESCRIPTION
**Pull request by vivekmenezes**
Release note: None

### CODE REVIEW

File: pkg/sql/schema_changer_test.go [hunk 1]
```diff
@@ -3241,15 +3241,94 @@ func TestCancelSchemaChange(t *testing.T) {
...
+		var x float64
+		if err := rows.Scan(&val, &x); err != nil {
+			t.Errorf("row %d scan failed: %s", count, err)
+			continue
+		}
+		if count != val {
+			t.Errorf("e = %d, v = %d", count, val)
+		}
+		if 1.2 != x {
+			t.Errorf("e = %f, v = %f", 1.4, x)
```
Comment (mjibson) [inline pkg/sql/schema_changer_test.go]: 1.4 -> 1.2

**Review by madelynnblue — APPROVED** [review]

**Comment by vivekmenezes** [comment]
TFTR!

**Pull request merged**